nuto/mechanics/mesh/MeshGmsh.cpp

```diff
@@ -97,9 +99,8 @@ GmshHeader ReadGmshHeader(std::ifstream& rFile)
     std::getline(rFile, line); // endl
     if (header.version < 2.)
     {
-        throw NuTo::Exception(__PRETTY_FUNCTION__,
-                              "Gmsh version 2.0 or higher requiered. - File version is " +
-                                      std::to_string(header.version));
+        throw NuTo::Exception(__PRETTY_FUNCTION__, "Gmsh version 2.0 or higher requiered. - File version is " +
+                                                           std::to_string(header.version));
```

Comment:
This now differs from the indentation produced by clang-format. So next time someone runs clang-format on this, we'll get the reverse change. Also: you could have at least fixed the typo (*requiered*) while you're messing with it ;)